src/Param.cc

```diff
@@ -440,6 +464,13 @@ bool Param::ValueFromString(const std::string &_value)
            << this->dataPtr->key << "].\n";
     return false;
   }
+  catch(const std::ios_base::failure &_e)
```

Comment:
nit Also possible to handle using `ss.fail()`?

Reply:
OK Per convo with Addisu, it's throwing exceptions, so nah, can't do this.

Reply:
Using `fail()` now because macOS doesn't throw an exception